qemu/tests/rng_driver_negative.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import aexpect
+
+from virttest import error_context
+from avocado.core import exceptions
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    KVM driver load test:
+    1) Log into a guest
+    2) Read from virtio-rng device
+    3) Unload the device driver
+    4) Check no crash
+    5) Load the device driver
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    login_timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=login_timeout)
+
+    driver_unload_cmd = params["driver_unload_cmd"]
+
+    error_context.context("Read virtio-rng device in background",
+                          logging.info)
+    read_rng_cmd = params["read_rng_cmd"]
+    status, pid = session.cmd_status_output(read_rng_cmd)
+
+    if status != 0:
```

Comment:
you can check it by status, status always 0 after execute your command. please check pid exits